src/caduceus/http.go

```diff
@@ -62,8 +64,8 @@ func (sh *ServerHandler) ServeHTTP(response http.ResponseWriter, request *http.R
 				response.Write([]byte(fmt.Sprintf("Only Content-Type values of \"application/json\" or " +
 					"\"application/msgpack\" are supported got: [%s].\n", value)))
 
-				debugLog.Log("explanation", "explanation","Only Content-Type values of \"application/json\" or " +
-					"\"application/msgpack\" are supported.", "actual", value)
+				errorLog.Log(messageKey,"Only Content-Type values 'application/json' or 'application/msgpack' are " +
```

Comment:
Same ^

Reply:
Again, the mapping should prevent content types that are unsupported from reaching this block of code.  Remove the log and verify that there's a `Headers("Content-Type", ...)` mapper.

Reply:
Addressed.